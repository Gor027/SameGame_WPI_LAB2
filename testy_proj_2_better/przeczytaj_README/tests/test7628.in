DD c   D c cDc D DDDDc   D  cc cD DDcDDcc D Dc Dc D
DD DDc c  D cc Dc  D cc  D DD D     DcD cDc cc
