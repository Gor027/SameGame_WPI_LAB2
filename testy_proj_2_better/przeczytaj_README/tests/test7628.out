  D   DcD   D c D c      cDD c DDc
ccccDcccDcDDDDccDDDcDcDDDDcDccDcccDcD
